xds/src/main/java/io/grpc/xds/XdsNameResolver.java

```diff
@@ -93,6 +96,8 @@
     this.backoffPolicyProvider = checkNotNull(backoffPolicyProvider, "backoffPolicyProvider");
     this.stopwatchSupplier = checkNotNull(stopwatchSupplier, "stopwatchSupplier");
     this.bootstrapper = checkNotNull(bootstrapper, "bootstrapper");
+    logger = XdsLogger.withLogId(InternalLogId.allocate("xds-resolver", null));
```

Comment:
You can pass `name` to the second arg of `allocate()`

Reply:
Done.